dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -270,4 +270,27 @@ LOGOUT_TOKEN_MISSING_CLAIMS=CWWKS1545E: The logout token is not valid because it
 LOGOUT_TOKEN_MISSING_CLAIMS.explanation=The logout token must contain the claims that are specified in the message.
 LOGOUT_TOKEN_MISSING_CLAIMS.useraction=Ensure that the OpenID Connect provider includes the missing claims in the logout token.
 
+# Do not translate "sub" or "sid"
+LOGOUT_TOKEN_MISSING_SUB_AND_SID=CWWKS1546E: The logout token is not valid because it does not contain a "sub" or "sid" claim. The token must have at least one of those claims.
+LOGOUT_TOKEN_MISSING_SUB_AND_SID.explanation=The sub and sid claims identify the user and session that is associated with a token. At least one of those claims must be present to determine the user or session to log out.
+LOGOUT_TOKEN_MISSING_SUB_AND_SID.useraction=Ensure that the OpenID Connect provider includes the missing claims in the logout token.
+
+# Do not translate "events"
+LOGOUT_TOKEN_EVENTS_CLAIM_WRONG_TYPE=CWWKS1547E: The logout token is not valid because the "events" claim is not a JSON object: {0}
+LOGOUT_TOKEN_EVENTS_CLAIM_WRONG_TYPE.explanation=The events claim in the logout token must be a JSON object.
+LOGOUT_TOKEN_EVENTS_CLAIM_WRONG_TYPE.useraction=Ensure that the OpenID Connect provider uses the correct data type for the events claim.
+
+# Do not translate "events"
+LOGOUT_TOKEN_EVENTS_CLAIM_MISSING_EXPECTED_MEMBER=CWWKS1548E: The logout token is not valid because the "events" claim does not contain "{0}" as a member. The "events" claim value is {1}.
```

Comment:
when the variable is resolved, is it still wrapped in {curly brackets}? If so, I don't think you need the quotes, if not, ok as is

Reply:
No, the resolved variable will not be wrapped in curly brackets. The resolved value will be a literal string that we expect to find (e.g. `CWWKS1548E: The logout token is not valid because the "events" claim does not contain "this key" as a member...`).